src/ophys_etl/transforms/data_loaders.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+import numpy as np
+from collections import namedtuple
+
+motion_border = namedtuple('motion_border', 'left right up down')
+
+
+def get_max_correction_values(motion_corr_df: pd.DataFrame,
+                              max_shift: float = 30.0) -> motion_border:
+    """
+    Gets the max correction values in the cardinal directions from
+    a motion correction dataframe created during Motion Correction workflow
+    step.
+    Parameters
+    ----------
+    motion_corr_df: pd.DataFrame
```

Comment:
Rather than passing a dataframe expecting particular columns, it would be clearer in this case to ask for the required series (since they're only two, and they're coordinates) -- in this case, the series of 'x' and 'y' cartesian coordinates (I believe? The docstring is not clear).

Reply:
Eliminated column checking and moved to expecting pandas.Series objects